packages/ui-kit/src/design-system.ts

```diff
@@ -7,9 +7,12 @@ const rootFontSizeNumber = 16;
 // must float / stretch into the available width
 const pageWidth = '770px';
 const pageWidthSmall = '592px';
+const pageWidthSmallReleaseNotes = '474px';
+const pageWidthReleaseNotes = '670px';
 const asideColumnWidth = pageWidthSmall;
 const navbarWidth = '224px';
 const navbarWidthSmall = '200px';
+const navbarWidthReleaseNotes = '324px';
```

Comment:
This is the width that we need for the index nav in order to show the date picker in it's full width.